app/views/catalog/_did_you_mean.html.erb

```diff
@@ -1,5 +0,0 @@
-<% if should_show_spellcheck_suggestions? @response and  !@response.spelling.collation.nil? %>
```

Comment:
This partial is no longer needed, since we don't use the "Did you Mean?" feature